troubleshooting.md

```diff
@@ -149,17 +149,45 @@ the [networking document](networking.md). The IPs and ports of the Ceph
 cluster's monitors should be in `/etc/ceph.conf`. From within the pod, read
 this and try to reach the IP as well as connect to the port.
 ```
-$ oc debug --container glance-api pod/glance-default-external-api-0
-bash-5.1$ cat /etc/ceph/ceph.conf
-# minimal ceph.conf for 7fbaf2fd-f70e-5625-b00e-65b33ee823a6
+$ oc get pods | grep glance | grep external-api-0
```

Comment:
I think we can remove `| grep glance` as we're going to get the same `Pod` in the example

Reply:
done